app/models/recommendation.py

```diff
@@ -128,6 +130,6 @@ async def __thompson_sample(slate_id: str, recommendations: ['RecommendationMode
             click_data = await ClickdataModel.get(slate_id, item_ids)
         except ValueError:
             rec_item_ids = ','.join(item_ids)
-            print(f'click data not found for candidates with item ids: {rec_item_ids}')
+            logging.warn(f'No click data found for {slate_id = } {rec_item_ids = }')
```

Comment:
What does the `= }` do? I couldn't immediately find the meaning in the [f-string spec](https://www.python.org/dev/peps/pep-0498/#specification).
```python
f ' <text> { <expression> <optional !s, !r, or !a> <optional : format specifier> } <text> ... '
```

Reply:
It prints out the variable name and its value:
```
>>> a = "corgis are the longest bois"
>>> print(f"{a =}")
a ='corgis are the longest bois'
```